### ISSUE TITLE
Failed to regenerate Proposal after Agreement

### ISSUE THREAD

**Issue by ederenn** [golemfactory/yagna-triage#120]
Name: ederenn
OS+lang+version (if applicable): Windows 10, py, 3.8.6 py

```
[2021-06-01T12:43:05.853+0200 WARN  ya_market::negotiation::common] Failed to regenerate Proposal after Agreement [R-92c818e534d981f57ebe81a2be0669f92679e69d40ff1054043974cac5f8631f] termination. Failed regenerate proposal: Demand [137afc5e53d64ce594dad927181f342f-6c082a8cfe26c50f13b8116e4d8e8ed2e9b3dd6704444ebf444434cdd3bea0ba] not found..
```

**Comment by tworec** [comment]
@nieznanysprawiciel we need not emit such a `warn`. This is rather `debug` or even `trace`. We should cease trying emitting Proposal for nonexistent Demand or Offer.

**Comment by nieznanysprawiciel** [comment]
We should distinguish between situation when Offer/Demand is unsubscribed/expired and situation, when error happened on database level

**Comment by tworec** [comment]
closing via https://github.com/golemfactory/yagna/pull/1388

**Comment by maaktweluit** [comment]
We seen this again in the testing session today:
```
[2021-06-10T12:54:42.843+0200 WARN  ya_market::negotiation::common] Failed to regenerate Proposal after Agreement [R-500db9db13b518555914cda28b6c038d61a1426a9b9900972e8a803ee0741a6e] termination. Failed regenerate proposal: Offer [5177de4e0f474474aa94b3de5cf0a245-b2ee3ac3d5fcf6cea07d9afcadf13f004734a60cbe8ffd6d4c80075e0179349e] expired..```

**Comment by nieznanysprawiciel** [comment]
Resolved by: https://github.com/golemfactory/yagna/pull/1388

**Comment by tworec** [comment]
> Resolved by: [golemfactory/yagna#1388](https://github.com/golemfactory/yagna/pull/1388)

I did already closed it via #1388 , but @maaktweluit has reopened it...

**Comment by nieznanysprawiciel** [comment]
I'm closing since I can't reproduce and warning doesn't appear in my tests.
Please reopen if anyone notice this again
